File: app/src/main/java/com/riders/thelab/data/remote/dto/weather/FeelsLike.java
package com.riders.thelab.data.remote.dto.weather;

import com.squareup.moshi.Json;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

@Getter
@Setter
@ToString
public class FeelsLike {
    @Json(name = "day")
    private double day;
    @Json(name = "night")
    private double night;
    @Json(name = "eve")
    private double evening;
    @Json(name = "morn")
    private double morning;
}

File: app/src/main/java/com/riders/thelab/data/remote/dto/weather/FeelsLike.kt
package com.riders.thelab.data.remote.dto.weather

import com.squareup.moshi.Json
import com.squareup.moshi.JsonClass
import kotlinx.parcelize.Parcelize

@JsonClass(generateAdapter = true)
data class FeelsLike(
    @Json(name = "day")
    var day: Double = 0.0,
    @Json(name = "night")
    val night: Double = 0.0,
    @Json(name = "eve")
    val evening: Double = 0.0,
    @Json(name = "morn")
    val morning: Double = 0.0
)
